### ISSUE TITLE
Need of an exact threshold point

### ISSUE THREAD

**Issue by pssaha** [qupath/qupath#237]
Hello, 

we are doing an exciting research back in USA, using your software. We all loved this masterpiece. 

Anyway, my question is simple. I attached a screenshot for your help. 

First, I used DAB channel filter my original image. 
Second, I clicked on Auto. 

Now, what I am struggling with is finding out the 'exact intensity value' of the Minimum display. I need this exact figure to set up a cutoff threshold to identify the total number of nuclei.

I'll appreciate if you kindly help us!

Kind regards,
Partha, An Lab, South Dakota State University, Brookings, USA

![sc](https://user-images.githubusercontent.com/44507241/47583089-fc988b00-d91b-11e8-95ee-587a897ee87d.JPG)

**Comment by petebankhead** [comment]
Hi @pssaha - great to hear about the exciting research!
Try double-clicking where it says 'Min display'.  Also, if this is for a positive cutoff (rather than detection), see https://petebankhead.github.io/qupath/tips/2018/03/22/setting-positive.html

**Comment by pssaha** [comment]
Hello, 

We are doing an exciting research back in USA, using your software. We all loved this masterpiece. 

Anyway, my question is simple. I attached a screenshot for your help. 

First, I used DAB channel filter my original image. 
Second, I clicked on Auto. 

Now, what I am struggling with is finding out the 'exact intensity value' of the **Minimum display**. I need this exact figure to set up a cutoff threshold to identify the total number of nuclei.

I'll appreciate if you kindly help us!

Kind regards,
Partha, An Lab, South Dakota State University, Brookings, USA

![sc](https://user-images.githubusercontent.com/44507241/47583396-f2c35780-d91c-11e8-82a8-e4b6be30a2c2.JPG)
